assets/js/upchunk_upload.js

```diff
@@ -36,8 +36,8 @@ const uploadInitializer = (entry, onViewError) => {
     upload.on("progress", (e) => {
       const now = new Date().getTime()
 
-      if (!window.uploaderReorderInProgress && !upload.paused && e.detail < 100 && now - lastProgressUpdate > 1000) {
-        entry.progress(e.detail)
+      if (!window.uploaderReorderInProgress && upload.success && !upload.paused && e.detail < 100 && now - lastProgressUpdate > 1000) {
+        console.log("progress", e.detail)
```

Comment:
is `entry.progress` the same as `console.log` ?

Reply:
You are replacing one with another. Is it ok ?

Reply:
fixed